.idea/misc.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="..\:/Users/compu/AndroidStudioProjects/chuck-norris-facts-android/app/src/main/res/layout/activity_main.xml" value="0.22826086956521738" />
```

Comment:
This key is specific to your local machine. We may want to add this entire xml file to the .gitignore